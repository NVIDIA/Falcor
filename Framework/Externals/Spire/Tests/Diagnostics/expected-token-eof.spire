// expected one token, but got EOF

int foo()
{
	int a = 3