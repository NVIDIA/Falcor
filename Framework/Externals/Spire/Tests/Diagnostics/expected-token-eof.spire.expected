result code = -1
standard error = {
Tests/Diagnostics/expected-token-eof.spire(0): error 20001: unexpected end of file, expected ';'
}
standard output = {
}
